data/txHandler.go

```diff
@@ -834,52 +855,33 @@ func (handler *TxHandler) validateIncomingTxMessage(rawmsg network.IncomingMessa
 	wi := &txBacklogMsg{
 		rawmsg:                &rawmsg,
 		unverifiedTxGroup:     unverifiedTxGroup,
-		rawmsgDataHash:        msgKey,
 		unverifiedTxGroupHash: canonicalKey,
 		capguard:              nil,
+		syncCh:                make(chan network.ForwardingPolicy, 1),
 	}
 
-	if handler.checkAlreadyCommitted(wi) {
```

Comment:
So we are cutting down on duplicated logic between what the backlogWorker is doing vs incoming message validation.

Reply:
`checkAlreadyCommitted` is part of `backlogWorker` that is called for p2p handler now.